BPRBlazor/Components/Results/Charts/PieChart.razor

```diff
@@ -0,0 +1,17 @@
+﻿@using Radzen.Blazor
+
+<div class="d-flex justify-content-center my-4">
+    <RadzenChart Style="height: 350px; width: 350px">
+        <RadzenPieSeries 
+            Radius="150" 
+            Fills=@(new [] {"#6f13a9", "#9b83af", "#955cb8"}) 
```

Comment:
What happens if you have data for 4 colors? Same with BarChart. It only supports 1 color. What if you have 2 series?